app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -177,7 +127,7 @@ If you're planning on training to teach languages and physics, you could be elig
 
 If you are not training to teach in these subjects, any support you get will depend on your immigration status. [Learn more about fees and funding for non-UK trainees](/non-uk-teachers/fees-and-funding-for-non-uk-trainees).
 
-## Search and apply for teacher training courses
+## Find and apply for teacher training courses
 
 Your teacher training course must lead to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). Some courses combine QTS with a [postgraduate certificate in education (PGCE) or postgraduate diploma in education (PGDE)](/train-to-be-a-teacher/what-is-a-pgce). These are optional: you do not need a PGCE or PGDE to teach in England.
```

Comment:
@laurahermionetennant92 not sure if this is right that it must lead to QTS? Some courses are PGCE without QTS but you need QTS to teach in maintained schools in England and you get all the extra support as an ECT?

Reply:
I see what you meant that there are some schools were you don't need QTS – however, choosing a course which doesn't lead to QTS would massively disadvantage them in every sector of the teaching job market. It would be a mad thing to do.